status-im/status-react
# Allow admin to leave chat

### Description

partially covers: #4868 (only for group chats)

1) Allow admin to leave a chat (should not receive anymore messages)
2) Allow admin to make other members admin
3) Display system messages when a user is made admin

### Testing

To make a member an admin:
1) Create a group chat
2) Go to Group-Info -> Click on Dots -> Make admin

If the user is already an admin, you will only see "View profile"

When leaving a chat, if you are an admin, you will be removed from the chat and will not receive any more messages (previous version will still show the user as admin).

status: ready
